### PR TITLE
add Capture mode signal to Panda DataBlock

### PR DESCRIPTION
**Pull request by jsouter**
Addresses https://github.com/bluesky/ophyd-async/issues/471

Implemented CaptureMode as an int enum to match what's in PandABlocks-ioc

### CODE REVIEW

**Comment by jwlodek** [comment]
Should we make the enum use the string names for each value, instead of int? Like with https://github.com/bluesky/ophyd-async/blob/1559aa0e451ffc93da46f5a0fdc7cf282def7fab/src/ophyd_async/epics/adcore/_core_io.py#L71

**Comment by jsouter** [comment]
> Should we make the enum use the string names for each value, instead of int? Like with
> 
> https://github.com/bluesky/ophyd-async/blob/1559aa0e451ffc93da46f5a0fdc7cf282def7fab/src/ophyd_async/epics/adcore/_core_io.py#L71

Would require a small change to the IOC but that would probably be nicer

File: src/ophyd_async/fastcs/panda/_block.py [hunk 1]
```diff
@@ -7,6 +7,12 @@
 from ._table import DatasetTable, SeqTable
 
 
+class CaptureMode(int, Enum):
+    first_n = 0
+    last_n = 1
+    forever = 2
```
Comment (evalott100) [inline src/ophyd_async/fastcs/panda/_block.py:13]:
```suggestion
class CaptureMode(str, Enum):
    FIRST_N = "FIRST_N"
    LAST_N = "LAST_N"
    FOREVER = "FOREVER"
```

**Review by evalott100 — DISMISSED** [review]

**Comment by evalott100** [comment]
> > Should we make the enum use the string names for each value, instead of int? Like with
> > https://github.com/bluesky/ophyd-async/blob/1559aa0e451ffc93da46f5a0fdc7cf282def7fab/src/ophyd_async/epics/adcore/_core_io.py#L71
> 
> Would require a small change to the IOC but that would probably be nicer

I everythings already sorted on the pandablocks-ioc and ophyd async epics backend for this.

**Pull request merged**